modules/ROOT/pages/database-administration/standard-databases/listing-databases.adoc

```diff
@@ -171,6 +181,8 @@ The value is a string formatted as:
 ----
 A database must be `online` or `deallocating` for this value to be available.
 For other database states the value will be `NULL`.
+
+The value for composite databases is `NULL` because it does not apply to them.
```

Comment:
```suggestion
The value for composite databases is `NULL` because it does not apply to them.footnote:compositeDb[]
```
so you added the footnote to some that wasn't updated and missed one of the ones that were updated